README.asciidoc

```diff
@@ -190,6 +190,12 @@ The motivation for this came up from the http://www.jhug.gr/[Greek Java User Gro
 * Description: The Prometheus monitoring system and time series database. https://prometheus.io/
 * What it means: A name from Greek Mythology, of a Titan, who stole the fire from the gods and gave it to humanity. https://en.wikipedia.org/wiki/Prometheus
 
+=== Pyrsia
+
+* Site: https://github.com/pyrsia/pyrsia
+* Description: Decentralized Package Network
```

Comment:
I know they use "Network" on their site, but a "Decentralized Package Network" makes very little sense to me and it makes me think of network packages... I think "Package Repository" is a better understood term, on the other hand. 

```suggestion
* Description: Decentralized Package Repository
```